src/loaders/tiffPyramidLoader.js

```diff
@@ -56,6 +58,13 @@ export default class TiffPyramidLoader {
     return rasters;
   }
 
+  getRasterSize({ z }) {
+    const image = this.channelPyramids[0][z];
+    const imageWidth = image.getWidth();
+    const imageHeight = image.getHeight();
+    return { imageWidth, imageHeight };
```

Comment:
Maybe just `height` and `width` unless that would overload something?  Would make https://github.com/hubmapconsortium/vitessce-image-viewer/pull/135/files#diff-c4254ec207f94571630e0c6e3b6528a9R93-R94 nicer

Reply:
see latest commit.